lib/src/source/git.dart

```diff
@@ -40,6 +41,49 @@ class GitSource extends CachedSource {
     });
   }
 
+  /// Gets the list of all versions from git tags.
+  /// Anything that Version.parse understands is considered a version,
+  /// it will also attempt to strip off a preceding 'v' e.g. v1.0.0
+  Future<List<Pubspec>> getVersions(String name, description) async {
+    if (!_useVersionTags(description)) {
```

Comment:
It'd be possible to access `version` if it was in the description under `git`, like you initially had it, right?
```yaml
dependencies:
  w_flux:
    version: "^1.0.0"
    git: git@github.com:Workiva/w_flux.git
```

Is there any reason that version can't be under `git`?